.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Test with pytest
+
+# Controls when the workflow will run
+on:
+  push:
+    branches: 
+    - '*'
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
```

Comment:
Please update to `actions/checkout@v2.3.4`, same as in `build.yml`.
Dependabot is set up for this repo and will also workflows.

Otherwise LGTM.

Reply:
Good catch! 
Just saw that dependabot also support git actions.